### ISSUE TITLE
Do not validate taxids when using -onlydesc in rfci.pl

### ISSUE THREAD

**Issue by AntonPetrov** [Rfam/rfam-family-pipeline#70]
The following command

```
rfci.pl -onlydesc -m 'Fix characters in AU line' RF04184
```

resulted in an error:

```
ERROR trying to fetch taxids from genbank, reached maximum allowed number of failed attempts (10)
```

The family was successfully checked in after a few attempts, but in general `-onlydesc` should have prevented the taxid check.

Seems like the problem is in `_commitEntry` of `Commit.pm`.

More logs:

```
rfci.pl -onlydesc -m 'Fix characters in AU line' RF04184
Successfully loaded local copy of RF04184 through middleware
Successfully loaded SVN copy of RF04184 through middleware
No GO mappings for this family-have you tried to add any?
Look up:
http://www.geneontology.org/ or  http://www.ebi.ac.uk/QuickGO/
Failed to commit family, RF04184: [A repository hook failed: Commit failed (details follow):: Commit blocked by pre-commit hook (exit code 1) with output:
[
    [0] "trunk/Families/RF04184/DESC"
]
trunk/Families/RF04184/DESC[]
[]
ERROR trying to fetch taxids from genbank, reached maximum allowed number of failed attempts (10)
Tried to fetch taxid
-
at rfam-family-pipeline/Rfam/Lib/Bio/Rfam/FamilyIO.pm line 4156.
DBIx::Class::Storage::TxnScopeGuard::DESTROY(): A DBIx::Class::Storage::TxnScopeGuard went out of scope without explicit commit or error. Rolling back. at perl5/lib/perl5/Carp.pm line 291
 at rfam-family-pipeline/Rfam/Lib/Bio/Rfam/SVN/Client.pm line 597.
]
 at rfam-family-pipeline/Rfam/Lib/Bio/Rfam/SVN/Client.pm line 599.
	Bio::Rfam::SVN::Client::commitFamilyDESC(Bio::Rfam::SVN::Client=HASH(0x1cf17c8), "RF04184") called at rfam-family-pipeline/Rfam/Scripts/svn/rfci.pl line 187
